pipelines/_ingredients/DECam/DRP.yaml

```diff
@@ -46,24 +46,21 @@ tasks:
       background_provider: "input_summary"
       connections.photoCalibName: "jointcal"
       connections.skyWcsName: "jointcal"
-  makeDirectWarp:
-    class: lsst.pipe.tasks.make_direct_warp.MakeDirectWarpTask
+  makeWarp:
+    class: lsst.pipe.tasks.makeWarp.MakeWarpTask
     config:
       # These configs match obs_subaru, to facilitate 1:1 comparisons between DECam and HSC
-      warper.warpingKernelName: lanczos5
+      matchingKernelSize: 29
+      modelPsf.defaultFwhm: 7.7
```

Comment:
I am not sure what the correct values are here, so I will only suggest double-checking them.